packages/cli/src/databases/migrations/postgresdb/1691088862123-CreateWorkflowNameIndex.ts

```diff
@@ -0,0 +1,15 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+export class CreateWorkflowNameIndex1691088862123 implements ReversibleMigration {
+	indexName = '48f450ec4a8536fbd7d8e7d094';
```

Comment:
why aren't we using human readable names for indexes?

Reply:
What's the convention for index names? We seem to be using UUIDs or so I thought.

Reply:
we usually have had people create indices manually on the DB, and then copy the auto-generated index name. but, we don't need to continue doing it that way.
we could also update `createIndex` to auto-generate more readable names automatically from the table and column names.